README.md

```diff
@@ -0,0 +1,235 @@
+
+<!-- ## Funtion 1 inputQuestionTemplate
+* ***Functionality***: Implement the question template
+* ***Input***:
+    * templateId: the id of the template
+    * template: template represented by a list
+    <br>e.g. "What is xxx" is represented by ["What is ", 0]
+    <br>e.g. "xxx's history" is represented by [0, "'s history"]
+* ***Output***: 0 for success and -1 for failed
+```python
+def inputQuestionTemplate( templateId, template ):
+   xxxxxx
+   return 0
+``` -->
+
+<!-- ## Function 2 getGoogleResults
+* ***Functionality***: Get searching results from Google Search with question
+* ***Input***:
+   * question: the question to be searched on Google
+   * resNum: the number of results as original text for summarization
+   * conditionId: potential condition for Google Search result filter
+   <br>e.g. if the conditions are pre-defined with specific id for each one, such as only with “.edu“ websites, then the source of original text will only be ".edu" websites
+* ***Output***: Original text stored in a list of length resNum, with each element from one piece of Google Search result
+```python
+def getGoogleResults( question, resNum, conditionId ):
+   xxxxxx
+   return originalText
+``` -->
+
+## Module 1: Data Collection
+### Function 1 getGoogleResults
+* ***Functionality***: Get searching results from Google Search with question
+* ***Input***:
+   * keywords: the keywords for the question
+   * resNum: the number of results as original text for summarization
+   <br>e.g. if the conditions are pre-defined with specific id for each one, such as only with “.edu“ websites, then the source of original text will only be ".edu" websites
+* ***Output***: Original text stored in a list of length resNum, with each element from one piece of Google Search result
+```python
+def getGoogleResults( keywords, resNum ):
+   xxxxxx
+   return originalText
+```
+
+<!-- ### Function 2 optimizeResultsWithConcept
+* ***Functionality***: Select the best results from Google based on the key concept
+* * ***Input***:
+   * keywords: the keywords for key concepts for the question
+   * resNum: the number of optimized results
+   * originalText: the original text to be optimized, stored as a list
+* ***Output***: Optimized text stored in a list of length resNum
+```python
+def optimizeResultsWithConcept( keywords, resNum, originalText ):
+   xxxxxx
+   return optimizedText
+``` -->
+
+## Module 2: Model Training (if necessary for the user)
+### Function 2 trainModel
+* ***Functionality***: Train the model if it is necessary for the user
+* ***Input***:
+   * referenceText: standard reference text is needed for calculation of saliency scores
+   * data: data with sentences from original text
+   * word2vecWordNum: number of words used in word embedding with word2vec
+   * wordNum: maximum number of words to keep (with relatively large frequency)
+* ***Output***: trained model
+```python
+def trainModel( referenceText, data, word2vecWordNum, wordNum ):
```

Comment:
Could you provide add more detail / clarification of what the `data` argument? Is it the training data?

Reply:
I have clarified the data argument as the training data. Thanks. I wonder if I should merge the request by myself after your second review, or you merge it after reviewing my correction?